### PR TITLE
Merge release 0.3.3 changes to develop

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -59,11 +61,15 @@ accessed directory. examples/cookbook/recipe_coalesce_pool.cpp shows how to do
...
-- Bug in ResourceManager::copy/memset when given a pointer offset into an allocation.
+
+- Bug in ResourceManager::copy/memset when given a pointer offset into an
+allocation.
 
 - Memory leak in judyL2Array.
 
-- While replay already was recording release operations, the tool was not actually replaying them.  A fix was implemented so that the replay tool will now also replay any captured release operations.
+- While replay already was recording release operations, the tool was not
+actually replaying them.  A fix was implemented so that the replay tool will
```
Comment (mcfadden8) [inline CHANGELOG.md]: What caused these changes?

Comment (davidbeckingsale) [reply CHANGELOG.md]: I line wrapped stuff correctly.

Comment (mcfadden8) [reply CHANGELOG.md]: Hmm.  I assumed that since this was markdown that line wrapping was taken care of.  It seems to render fine in either case.

Comment (mcfadden8) [reply CHANGELOG.md]: What is the "correct" line wrap?  80 columns?  I'll try to remember that for the next time.

**Pull request merged**